### ISSUE TITLE
NTR: glycogen granule-autophagosome adaptor activity

### ISSUE THREAD

**Issue by Antonialock** [geneontology/go-ontology#28283]
Please provide as much information as you can: 

* **Suggested term label:**

glycogen granule-autophagosome adaptor activity

* **Definition (free text)**
The binding activity of a molecule that brings together a glycogen granule and an autophagosome during glycophagy, the autophagy of glycogen. 

* **Reference, in format PMID:#######**
 38832010

* **Gene product name and ID to be annotated to this term**

atg45 YIL024C

* **Parent term(s)**

GO:0160183

* **Any other information**

in https://www.sciencedirect.com/science/article/pii/S2589004224010320?via%3Dihub
the authors term atg45 an autophagy receptor for glycogen 

![image](https://github.com/geneontology/go-ontology/assets/12727506/4650ae24-1136-4f09-bb6e-3c38ff76d7a7)

section: **Atg45 functions as a glycogen receptor**
...Atg45 binds to glycogen through the GBD identified within Atg45 1-96
...the C-terminal half of Atg45 is required for sequestration into autophagosomes,
...These results suggest that an AIM is present within the C-terminal half of Atg45. We identified 127YVNL130 as the sole candidate for an AIM within the region. To investigate whether this site functions as AIM, we carried out co-immunoprecipitation of GFP-Atg8 and Atg45-FLAG using the GFP-Trap magnetic agarose system. While WT Atg45 exhibited binding to Atg8, the alanine-substituted mutant of the putative AIM (Y127A and L130A, herein Atg45AIMm) had a low binding ability

**Discussion:**
We found that Atg45 is important for glycophagy and that this protein binds to both glycogen and Atg8. We therefore conclude that Atg45 functions as a glycophagy receptor.

**Comment by pgaudet** [comment]
Hi @Antonialock 

Is this a receptor or an adaptor? Could this activity be a child of [cargo receptor activity](http://purl.obolibrary.org/obo/GO_0038024)? 

Thanks, Pascale

**Comment by Antonialock** [comment]
I don't think so? I think cargo receptor refers to endocytic things? (atg45 is also not transmembrane spanning)

doesn't GO:0160183 autophagosome-membrane adaptor activity look correct? ohh I suppose it doesn't link the autophagosome to a membrane, it links it to a macromolecular complex?

**Comment by Antonialock** [comment]
table 1 here might be of interest
https://febs.onlinelibrary.wiley.com/doi/full/10.1111/febs.15824
Table 1. Selective autophagy types and corresponding receptors.

**Comment by ValWood** [comment]
This is more of a sequestering activity, maybe?


GO:0140313   molecular sequestering activity

**Comment by pgaudet** [comment]
@marcfeuermann 
I notice that we we don't have a term for 'autophagy receptor'; is this missing in the ontology ? If not, how is this supposed to be captured?

**Comment by marcfeuermann** [comment]
There are tens of thousands of papers about autophagy and we could create hundreds of new terms about it. So, many terms are probably still missing, especially since it is a hot topic and everyone wants to find a new autophagy-related mechanism. 
About this specific request, personally, I wouldn't call these receptors. Can we talk about mitochondria or endoplasmic reticulum receptors ? It seems rather that these molecules act as adaptors between ATG8 proteins linked to the autophagosome membrane and macromolecules or organelles (via proteins present in the membranes of the organelles). I'd go for something along the lines of "cargo-autophagosome adaptor activity", as Antonia first suggested. After that, if you think you need a term for each type of autophagy, you can create all the corresponding children (glycophagy, aggrephagy, reticulophagy, mitophagy,  lipophagy, ...).
To add more fuel to my fire, some children terms already exist:
GO:0140580 mitochondrion autophagosome adaptor activity 
GO:0160183 autophagosome-membrane adaptor activity 

And we have the following related process term that should probably be obsoleted:
GO:0061753 substrate localization to autophagosome
I hope this will be helpful to take a decision.
Best regards,
Marc.

**Comment by ValWood** [comment]
I agree that "cargo receptor" sounds appropriate. We could broaden the definition from 

Binding specifically to a substance (cargo) and a membrane to deliver it to a transport vesicle [or to a vacuole]. PMID:15239958 PMID:27903609

This could be used for Antonia's example and we could also move
autophagosome-membrane adaptor activity (GO:0160183)
under  such term.

However,  we get in a mess because  currently
autophagosome-membrane adaptor activity (GO:0160183)
are types of 
protein-membrane adaptor activity (GO:0043495)
-- | --
The binding activity of a molecule that brings together a protein or a protein complex with a membrane, or bringing together two membranes, either via membrane lipid binding or by interacting with a membrane protein, to establish or maintain the localization of the protein, protein complex or organelle.

which seems to be more about normal location than a substrate for degradation.
so this definition would also need to b e broadened....

**Comment by raymond91125** [comment]
It seems to me that cargo receptor activity is intended for vesicle transport PMID:27903609, whereas here it's "cargo receptors, or selective autophagy receptors (SARs), which act as a bridge between the phagophore and cargo" PMIDcargo receptors, or selective autophagy receptors (SARs), which act as a bridge between the phagophore and cargo" PMID:38832010. The action of the adaptor here is prior to vesicle transport (the vesicle has not enclosed yet). I'm not sure it's a good idea to mix the two.

**Comment by pgaudet** [comment]
@raymond91125 

Thanks for the observations. Do you have a proposal to make, or questions for @hattrill ? I am not clear how you want to proceed, or what you need. 

Thanks, Pascale

**Comment by raymond91125** [comment]
+[Term]
+id: GO:0160247
+name: autophagy cargo adaptor activity
+namespace: molecular_function
+def: "The binding activity of a molecule that brings together a cargo, targeted for degradation via autophagy, to a phagophore." [PMID:23545414, PMID:31585693, PMID:36538890]
+synonym: "autophagy adaptor activity" EXACT []
+synonym: "selective autophagy receptor activity" EXACT []
+is_a: GO:0030674 ! protein-macromolecule adaptor activity
+relationship: part_of GO:0061753 ! substrate localization to autophagosome
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/28283" xsd:anyURI
+created_by: rynl
+creation_date: 2025-02-10T16:57:01Z

We then have:
protein-macromolecule adaptor activity
... autophagy cargo adaptor activity
...... autophagosome-membrane adaptor activity
......... endoplasmic reticulum-autophagosome adaptor activity
......... mitochondrion autophagosome adaptor activity

@Antonialock Please let me know if you'd prefer to have "glycogen granule-autophagosome adaptor activity" rather than specifying primary inputs during annotation to "autophagy cargo adaptor activity". Thanks.

**Comment by raymond91125** [comment]
Closed for now. Please reopen if needed. Thanks.
